src/ImageSharp/Formats/Jpg/JpegDecoderCore.cs

```diff
@@ -559,42 +547,42 @@ private void ProcessDefineHuffmanTablesMarker(int remaining)
             }
         }
 
-        private void ProcessDefineHuffmanTablesMarkerLoop(ref Huffman huffman, ref int remaining)
+        private void ProcessDefineHuffmanTablesMarkerLoop(ref HuffmanTree huffmanTree, ref int remaining)
```

Comment:
Ohh it is a struct, did not see that coming! That makes sense, don't mind me then :smile: